scripts/nix-profile.sh.in

```diff
@@ -1,4 +1,6 @@
 if [ -n "$HOME" ] && [ -n "$USER" ]; then
+    # __safe: if it is safe to "activate" Nix. 0 is safe, 1 is unsafe
+    __safe=0
```

Comment:
That’s confusing — should the variable be called __unsafe if the values are that way around?

I see that this has been flipped around a few times over the course of this patchset…

Reply:
Yes, I was just about to comment about this.

The condition for "is this unsafe?"  is a bit complicated. I found trying to flip the logic to be fraught, and I wasn't sure it was correct anymore.

If you can help with this, that would be great.

One note: this profile script gets executed at the start of _every_ incantation, and _must not fail_.

Reply:
You could call it `__safety` and use strings: `safe` and `unsafe`. Thus the "is this safe?" check is the relatively easy-to-digest expression `[ "$__safety" == safe ]`